presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -156,17 +156,31 @@ public void testIsDistinctFrom()
     }
 
     @Test
-    public void testRowEquality()
+    public void testRowComparison()
             throws Exception
     {
         assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') = " +
```

Comment:
hmmmm.... why is that? Looks like the strings are exactly the same on both side of '='